pytorch_lightning/logging/sacred.py

```diff
@@ -0,0 +1,69 @@
+from logging import getLogger
+from time import time
+
+try:
+    import sacred
+    from sacred.observers import MongoObserver
+except ImportError:
+    raise ImportError('Missing sacred package.  Run `pip install sacred`')
+
+from pytorch_lightning.logging.base import LightningLoggerBase, rank_zero_only
+
+logger = getLogger(__name__)
+
+
+# TODO: add docstring with type definition
+class SacredLogger(LightningLoggerBase):
+    def __init__(self, sacred_experiment, mongodb_settings):
+        super().__init__()
+        self.sacred_experiment = sacred_experiment
+        self.experiment_name = sacred_experiment.current_run.experiment_info["name"]
+        self._run_id = sacred_experiment.current_run._id
+
+        # for now we only support MongoObserver -> could be extended to be more flexible
+        self.sacred_experiment.observers.append(
+            MongoObserver.create(
+                url='mongodb://{ip}:{port}'.format(**mongodb_settings),
+                db_name='{db}'.format(**mongodb_settings),
+            )
+        )
```

Comment:
I think the best way to do this is to have the user create the sacred observer(s) on his own and avoid creating an additional observer in this constructor. This way, we support all observers supported by sacred, without committing to the mongodb one.

Reply:
Yes, makes a lot of sense. Then the user also has to take care of possible username/password for MongoDB for him/herself. At first I was only considering my usecase, but this will make it more useful/flexible.